pkg/cloudprovider/providers/vsphere/vsphere_util.go

```diff
@@ -532,7 +532,7 @@ func (vs *VSphere) checkDiskAttached(ctx context.Context, nodes []k8stypes.NodeN
 		if err != nil {
 			return nodesToRetry, err
 		}
-		glog.V(9).Infof("Verifying volume for nodeName: %q with nodeuuid: %s", nodeName, node.Status.NodeInfo.SystemUUID, vmMoMap)
+		glog.V(9).Infof("Verifying volume for %s: %q with nodeuuid: %s", nodeName, node.Status.NodeInfo.SystemUUID, vmMoMap)
```

Comment:
This still doesn't look right.  I think you need something like:

```
glog.V(9).Infof("Verifying volume for node %s with nodeuuid %q: %s", nodeName, node.Status.NodeInfo.SystemUUID, vmMoMap)

?

Reply:
@quinton-hoole  Done,PTAL